src/scripts/helpers/backbone/views/base.coffee

```diff
@@ -77,11 +77,12 @@ define (require) ->
         historyPage = linksHelper.getCurrentPathComponents().page
 
       if @pageTitle
-        document.title = @pageTitle + settings.titleSuffix
+        pageTitle = @pageTitle.replace(/<\/?[^>]+(>|$)/g, '') # Strip tags for baked HTML titles
+        document.title = pageTitle + settings.titleSuffix
```

Comment:
We have to trust the title coming from our DB: that's half the point of this. So no additional risk. But really, the browser executes scripts in an un-anchored div that is not part of the document?